examples/local/launch_cloud_onprem.py

```diff
@@ -0,0 +1,104 @@
+"""Automatically launches and sets up a Sky onprem cluster on AWS.
+
+The script allocates a cloud cluster and setups up the cloud cluster to run an
+'onprem' cluster on the cloud. It performs the following steps:
+    1) Launches a cluster on the cloud with `sky launch -c on-prem-cluster-[ID]`.
+     The Sky yaml sets up another user named `test`.
+    2) Setups Sky dependencies for the admin user via `sky admin deploy ..`.
+    3) Creates the local cluster config for the regular user, stored in `~/.sky/local/...`
+To clean up the local cluster, run `sky down [LOCAL_CLUSTER]` and remove the corresponding
```

Comment:
seems like `sky down [cluster_name]` is not sufficient - it just downs the onprem cluster, but not the actual VM running the cluster. For instance, I ran `python launch_cloud_onprem.py myclus`, but even after `sky down myclus` I have the host VM running:

```
(base) romilb@romilbx1yoga:/mnt/d/Romil/Berkeley/Research/sky-experiments/examples/onprem_test$ sky status
NAME                   LAUNCHED    RESOURCES            STATUS  AUTOSTOP  COMMAND                       
onprem-cluster-91644f  13 hrs ago  1x AWS(m6i.2xlarge)  UP      -         launch_cloud_onprem.py my...  
```

I believe we should fetch the autogenerated name add `sky down <name, eg. onprem-cluster-91644f>` to L103.

Reply:
Donwe, thanks!